Execute TSML fails since latest update.

We can no longer do a release from DevOps when using the Azure Analysis Service TMSL Script.

I get the following: Error during removing firewall rule (You cannot call a method on a null-valued expression.)

Please help?


Starting: executeaastsml
==============================================================================
Task         : Azure Analysis Service TMSL Script
Description  : Executes a TMSL Script on an Azure Analysis Service instance
Version      : 1.3.5
Author       : Jan Pieter Posthuma
Help         : [More Information](https://github.com/liprec/vsts-release-aas)
==============================================================================
Import-Module -Name C:\Modules\azurerm_2.1.0\AzureRM\2.1.0\AzureRM.psd1 -Global
##[warning]The names of some imported commands from the module 'AzureRM.Websites' include unapproved verbs that might make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
Clear-AzureRmContext -Scope Process
Disable-AzureRmContextAutosave -ErrorAction Stop
Add-AzureRMAccount -ServicePrincipal -Tenant *** -Credential System.Management.Automation.PSCredential -Environment AzureCloud @processScope
 Select-AzureRMSubscription -SubscriptionId ********* -TenantId ***
##[error]Error during removing firewall rule (You cannot call a method on a null-valued expression.)
##[error]You cannot call a method on a null-valued expression.
Finishing: executeaastsml

--- a/deploy-aas-db/v1/task.json
+++ b/deploy-aas-db/v1/task.json
@@ -16,7 +16,7 @@
     "version": {
         "Major": "1",
         "Minor": "3",
-        "Patch": "5"
+        "Patch": "6"
     },
     "minimumAgentVersion": "2.115.0",
     "groups": [

--- a/execute-aas-tmsl/v1/execute-aas-tmsl.psm1
+++ b/execute-aas-tmsl/v1/execute-aas-tmsl.psm1
@@ -255,7 +255,7 @@ An example
 .NOTES
 General notes
 #>
-function RemoveCurrentServerFromASFirewall($Server, $AzContext) {
+function RemoveCurrentServerFromASFirewall($Server, $AzContext, $Skip) {
     $serverName = $Server.Split('/')[3].Replace(':rw','');
     try {
         $currentConfig = (Get-AzureRmAnalysisServicesServer -Name $serverName -DefaultProfile $AzContext)[0].FirewallConfig
@@ -268,9 +268,11 @@ function RemoveCurrentServerFromASFirewall($Server, $AzContext) {
         }
         $result = Set-AzureRmAnalysisServicesServer -Name $serverName -FirewallConfig $firewallConfig -DefaultProfile $AzContext
     } catch {
-        $errMsg = $_.exception.message
-        Write-Host "##vso[task.logissue type=error;]Error during removing firewall rule ($errMsg)"
-        throw
+        if ($Skip -ne $true) {
+            $errMsg = $_.exception.message
+            Write-Host "##vso[task.logissue type=error;]Error during removing firewall rule ($errMsg)"
+            throw
+        }
     }
 
     return $true

--- a/execute-aas-tmsl/v1/task.json
+++ b/execute-aas-tmsl/v1/task.json
@@ -16,7 +16,7 @@
     "version": {
         "Major": "1",
         "Minor": "3",
-        "Patch": "5"
+        "Patch": "6"
     },
     "minimumAgentVersion": "2.115.0",
     "groups": [

--- a/vss-extension-preview.json
+++ b/vss-extension-preview.json
@@ -1,7 +1,7 @@
 {
     "manifestVersion": 1,
     "id": "vsts-release-aas-preview",
-    "version": "1.3.5",
+    "version": "1.3.6",
     "name": "Azure Analysis Service (Preview)",
     "publisher": "liprec",
     "public": false,

--- a/vss-extension.json
+++ b/vss-extension.json
@@ -1,7 +1,7 @@
 {
     "manifestVersion": 1,
     "id": "vsts-release-aas",
-    "version": "1.3.5",
+    "version": "1.3.6",
     "name": "Azure Analysis Service",
     "publisher": "liprec",
     "public": true,

